### PR TITLE
List project_details creado. 

### PR DESCRIPTION
**Pull request by koaladlt**
Dejé comentado el FK como hizo Ini, debido a dudas como sobre realizarlo.

### CODE REVIEW

File: lists/index.js [hunk 1]
```diff
@@ -1,9 +1,11 @@
 const Stuff = require('./stuff')
 const FAQ = require('./faq')
+const projectDetails = require('./projectDetail')
```
Comment (Vogric) [inline lists/index.js:6]: Maybe change projectDetails to ProjectDetails for better consistency with the rest of the lists? :)

The same in line 8.

Comment (koaladlt) [reply lists/index.js:6]: Done @Vogric

File: lists/projectDetail.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+                type: Url,
+            },
+            css_framework: {
+                type: Text
+            },
+            repository_link: {
+                type: Url,
+                isUnique: true
+            },
+            technical_description: {
```
Comment (Vogric) [inline lists/projectDetail.js:23]: Maybe we can add isMultiline here?

File: lists/projectDetail.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+const { Text, Url } = require('@keystonejs/fields');
+
+const ProjectDetails = (keystone) => {
+    keystone.createList('project_detail', {
+        fields: {
+            name: {
+                type: Text
+            },
+            project_description: {
```
Comment (Vogric) [inline lists/projectDetail.js]: `project_description` must be `product_description` according to the wireframe

image

**Review by faqndo97 — APPROVED** [review]

**Pull request merged**